spylon/spark/yarn_launcher.py

```diff
@@ -313,9 +321,18 @@ def launcher(deploy_mode, args, working_dir="."):
         run_pyspark_yarn_cluster(**args)
 
 
+def _fix_permissions(env_dir):
+    bin_dir = os.path.join(env_dir, "bin")
+    if os.path.isdir(bin_dir):
+        if platform.system() == 'Linux':
+            subprocess.check_call(["chmod", "-R", "a+x", bin_dir])
```

Comment:
err instead of `cmd`, it should be `["chmod", "-R", "a+x", bin_dir]`, but that's probably obvious